### PR TITLE
commits binary translator

### PR DESCRIPTION
**Pull request by mnhollandplum**

### CODE REVIEW

**Comment by mnhollandplum** [comment]
Nikki Holland-Plum

File: lib/binary_translator.rb [hunk 1]
```diff
@@ -29,4 +31,20 @@ def initialize
       "z" => "011010"
     }
   end
+
+def translated
+  @alpha_to_binary.values
+end
+
+def translates_a
```
Comment (memcmahon) [inline lib/binary_translator.rb:39]: Keep an eye on the interaction pattern!  This method should be #translate(argument).

File: lib/binary_translator.rb [hunk 2]
```diff
@@ -29,4 +31,20 @@ def initialize
...
+  @alpha_to_binary["a"]
+end
+
+def translates_z
+  @alpha_to_binary["z"]
+end
+def translates_turing
```
Comment (memcmahon) [inline lib/binary_translator.rb:46]: For all methods in this class, think about what a program should accomplish - it is meant to make our lives easier!  If we hardcode every value that we will ever want to translate, we are not creating less work for ourselves.

File: test/binary_translator_test.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+class BinaryTranslatorTest < Minitest::Test
+
+  def test_it_has_an_instance_of_Binary_Translator
+    bt = BinaryTranslator.new
+    assert_instance_of BinaryTranslator, bt
+  end
+
+  def test_it_translates_a
+    bt_a = BinaryTranslator.new
+    assert_equal "000001", bt_a.translates_a
```
Comment (memcmahon) [inline test/binary_translator_test.rb:15]: Your test structure looks really good, but remember to follow the interaction pattern and test the methods that are set out in the spec!

**Pull request closed without merging**